### PR TITLE
Feature/docker

### PR DESCRIPTION
**Pull request by giagermain**
dockerfile and docker-compose.yml created

### CODE REVIEW

**Comment by Eric-Butcher** [comment]
I believe Sumanyu needs to approve this one because the docker files are having issues with the project directory structure for Django.

**Comment by giagermain** [comment]
To elaborate: docker needs to be in the root directory or else it will throw the error "failed to solve: failed to read dockerfile: open Dockerfile: no such file or directory". After moving dockerfile and docker-compose.yml to the root directory, it then could not find manage.py since it also was not in the root directory with it. Since then all the directories were off, I pushed each of the files inside the inner evergreen folder to the outer evergreen folder and this solved the issue and docker ran properly.

Also, because there were some issues with my pull request that I didn't realize, there is now the original evergreen files in their initial folder location, and the evergreen files that i moved in their changed location. It seems like there are minor differences between the two "copycat" files" so I believe the files I used were outdated as well.

**Comment by sumanyuj** [comment]
1) In the docker compose file, under the command key, you can "cd" into the evergreen folder, which originally had the manage.py. This maintains the structure that I had, which is essentially default and something what I believe we should keep how it is. (So manage.py should go back into evergreen, and docker-compose should cd into evergreen dir before running runserver)

2) I am not able to run docker compose up from my machine. In VS Code, I get "services.web Additional property environment is not allowed" in regards to the docker-compose.yml file.

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: '3'
+services:
+  mongo:
+    image: mongo:4.2.5
+  web:
+    build: .
+    command: python manage.py runserver 0.0.0.0:8080
```
Comment (sumanyuj) [inline docker-compose.yml:7]: you can change the directory to be in the evergreen folder, which should have manage.py once you move it back there

File: requirements.txt
Comment (sumanyuj) [inline requirements.txt:1]: make sure to move this file to evergreen folder

**Review by sumanyuj — CHANGES_REQUESTED** [review]
Please see comment

File: manage.py
Comment (sumanyuj) [inline manage.py:1]: Make sure to move this file back to evergreen folder rather than root

**Review by sumanyuj — APPROVED** [review]
LGTM!

**Pull request merged**